### ISSUE TITLE
[suggestion]add command：after the download is complete, let "cmd" exit by itself.

### ISSUE THREAD

**Issue by dodolook8** [yt-dlp/yt-dlp#6535]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm requesting a feature unrelated to a specific site
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2023.03.04** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

add command：after the download is complete, let "cmd" exit by itself.

### Provide verbose output that clearly demonstrates the problem

- [ ] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [ ] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
add command：after the download is complete, let "cmd" exit by itself.
It's a bit of a hassle to click to close each time.
thx！

recommended by someone：
yt-dlp "URL" && exit

I want the command to appear in the form of "--" in front, so that I can copy it directly instead of typing "&& exit" every time.

otherone said：
--exec exit, if you want to add it to your config file

but i don't know how to modify config file.

In short, if add this command, it will be more convenient, and we don't need to enter "&& exit"  every time. 
thx!
```

**Comment by Lesmiscore** [comment]
well, while "just" closing the console after the command may be a hard task for some platforms, it _may be a good idea_ to take this as a request for running tasks after all videos finished downloading
still, using `&& exit` is an acceptable solution for your needs, though

**Comment by dirkf** [comment]
This was #6508?

**Comment by Lesmiscore** [comment]
bruh, OP duplicated the issue!

**Comment by dirkf** [comment]
After asking whether it was possible and being told not (within yt-dlp), this `suggestion` was created: but the old issue could be transformed.
